apps/ui/src/apis/rides/edit-ride.ts

```diff
@@ -0,0 +1,52 @@
+import { Ride } from '@communetypes/Ride';
+import { graphqlRequest } from '../graphql';
+import { EditRideSchema } from '@communetypes/EditRideSchema';
+import { updateRideQuery, updateRidersQuery } from '../utils/userRideQueries';
+import { Rider } from '@communetypes/Rider';
```

Comment:
why 3 imports from local package?

Reply:
What

Reply:
you have 3 different import from the types package

Reply:
So what? I need them